lib/segmentio/index.js

```diff
@@ -162,6 +162,7 @@ Segment.prototype.normalize = function(msg){
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.messageId = uuid();
+  msg.sentAt = new Date();
```

Comment:
ah got it, yeah we don't have local storage queue yet, but i think we might add one eventually.